emojivoto-web-app/js/setup_dev_env.sh

```diff
@@ -27,18 +30,28 @@ send_telemetry() {
     fi
 }
 
-has_cli() {
+display_step() {
```

Comment:
New output with highlighted steps:

```console
❯ ./setup_dev_env.sh
*** Step 1/7: Checking required tools ... Complete.
*** Step 2/7: Checking for AMBASSADOR_API_KEY environment variable
*** Step 3/7: Configuring development container. This container encapsulates all the dependencies needed to run the emojivoto-web-app locally.
This may take a few moments to download and start.
*** Step 4/7: Extracting KUBECONFIG from container
.
Listing services in emojivoto namespace
NAME             TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)             AGE
emoji-svc        ClusterIP   10.43.7.131     <none>        8080/TCP,8801/TCP   75m
voting-svc       ClusterIP   10.43.35.239    <none>        8080/TCP,8801/TCP   75m
web-svc          ClusterIP   10.43.134.106   <none>        8080/TCP            75m
web-app-canary   ClusterIP   10.43.26.66     <none>        80/TCP              75m
web-app          ClusterIP   10.43.236.162   <none>        80/TCP              75m
*** Step 5/7: Checking for Telepresence
Telepresence already installed
*** Step 6/7: Connecting local development environment to remote K8s cluster
Login successful.
Launching Telepresence Root Daemon
Connected to context default (https://demo-prod-55fb3ac1-a49e-4fe2-8cba-909af964ed78.kluster.bakerstreet.io)
Using ReplicaSet web-app-57bc7c4959
intercepted
    Intercept name         : web-app-57bc7c4959-emojivoto
    State                  : ACTIVE
    Workload kind          : ReplicaSet
    Destination            : 127.0.0.1:8083
    Service Port Identifier: 80
    Intercepting           : HTTP requests that match all headers:
      'x-telepresence-intercept-id: 6415be8f-06f4-466e-9f0d-61376cf34376:web-app-57bc7c4959-emojivoto'
    Preview URL            : https://elegant-mayer-8703.preview.edgestack.me
    Layer 5 Hostname       : ambassador.ambassador
*** Step 7/7: Opening editor

INSTRUCTIONS FOR DEVELOPMENT
============================
To set the correct Kubernetes context on this shell, please execute:
export KUBECONFIG=./emojivoto_k8s_context.yaml
```

Reply:
I think this is much easier to read -- WDYT?

Reply:
Yes I agree.
@joseldmc Can you run it on Mac please to be sure everything still work.